docs/guides/advanced/images.md

```diff
@@ -0,0 +1,106 @@
+
+Ingesting an image can take a few seconds, and Coda will show a paperclip icon for the image until it is complete. This delay can be more noticeable for sync tables, since they can return many images at once and they are ingested sequentially.
+
+!!! bug
+    Image attachments currently only work correctly within a sync table. When used outside of a sync table they behave like image references and load the image from the source URL.
+
+
+## Private images {: #private}
+
+Some APIs return links to private images, that require authentication credentials to be provided in order to access their content. Before Coda can ingest these images you must first copy them to a location that is publicly accessible. The Pack SDK provides a utility for this, known as [`TemporaryBlobStorage`][reference_temporaryblobstorage]. It is made available at `context.temporaryBlobStorage`, and the `storeUrl()` method fetches the image (with authentication credentials), stores the data in a temporary location, and returns the resulting URL. This temporary URL can then be returned as an `ImageAttachment` as [discussed above](#expiring).
+
+```ts
+let privateImageUrl = response.body.imageUrl;
+let temporaryImageUrl =
+  await context.temporaryBlobStorage.storeUrl(privateImageUrl);
+return temporaryImageUrl;
+```
+
+
+## Raw image data
+
+Some APIs return the raw image data directly in the response object, typically as a base64-encoded string. Additionally, JavaScript libraries used to generate or manipulate images may also return raw image data.
+
+Similar to [private images](#private) discussed above, you can use `TemporaryBlobStorage` to store the image data in a temporary location and get a URL which Coda can then ingest. The `storeBlob()` method accepts a [Node.js `Buffer`][buffer] containing the data and the content type of the image.
+
+```ts
+let imageBase64 = response.body.image;
+let buffer = Buffer.from(imageBase64, "base64");
+let temporaryImageUrl =
+  await context.temporaryBlobStorage.storeBlob(buffer, "image/png");
+return temporaryImageUrl;
+```
+
+
+## SVG images
+
+[Scalable Vector Graphics (SVG)][mdn_svg] is an XML-based markup language for drawing images, and Packs can generate SVGs to create custom visualizations. For example, the [Latex Pack][pack_latex] uses a JavaScript library to generate SVG images of math equations.
+
+Instead of storing SVG images in temporary blob storage, you can instead encode them to a base64 string and return them as a [Data URl][mdn_data_urls]
+
+```ts
+let svg = "...";
+let encoded = Buffer.from(svg).toString("base64");
+return coda.SvgConstants.DataUrlPrefix + encoded;
+```
+
+!!! warning
+    Data URLs should only be used for relatively small images, like SVGs. Returning large images as data URLs can cause performance issues in the document.
```

Comment:
I'd make this a really strong warning. It should only be used for "very" small images, and if you're returning large images or lots of inline images, it could eventually cause docs to break entirely.

Reply:
Done:
![Screen Shot 2022-02-22 at 12 35 01 PM](https://user-images.githubusercontent.com/88106038/155187094-fb71dc06-53f8-48f9-9fb7-01cd62ab2599.png)